### PR TITLE
Starman, the One and Only

### PR DESCRIPTION
**Pull request by ghost**
See demo here: https://webm.red/T4Vy.webm
Edit: you can hear me watching mason stream in background, whoops

Adds the starman for adminbus purposes. I'll probably work on it more, add it as an antag, and give it some items. Probably needs some kind of gun to pose a threat at all. Wanted to get this up though. It also does an edit to allow silicons to use targeted spells and updates their HUD.

Psi Starstorm Omega
Summons a fearsome starstorm around the starman - 15 mini meteors, each with a tiny explosion. 120 sec CD.

Psi Brainshock Omega
Shocks all organics in a radius around the starman, knocking them down for 1 second and doing some jittering and stuff. I probably should improve this one. 30 sec CD.

Psi Shield Beta
Generates an indestructible 3 tile-wide wall at the targeted location, oriented to protect you. Lasts 20 seconds, 15 sec CD.

Warp
The starman cannot be moved or move itself, but it has a 6 second CD targeted teleport. Can teleport into unseeable tiles. 

Telepathic Binaural Attack
Plays some sick Earthbound beats to frighten those stupid organics.

:cl:
 * rscadd: Added starman for adminbussing.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_subtypes/starman.dm [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+	
+/mob/living/silicon/robot/starman/wearing_wiz_garb()
+	return 1
+	
+/mob/living/silicon/robot/starman/New()
+	..()
+	UnlinkSelf()
+	laws = new /datum/ai_laws/starman()
+	pick_module(STARMAN_MODULE)
+	set_module_sprites(list("Basic" = "starman"))
```
Comment (jknpj) [inline code/modules/mob/living/silicon/robot/robot_subtypes/starman.dm:14]: No need if it only has one icon

**Comment by BomberBro** [comment]
>It also does an edit to allow silicons to use targeted spells and updates their HUD.

Should be its own PR tbh

**Comment by Kharkov72** [comment]
yeah just because even if this doesn't go through that would be nice to have

**Comment by ghost** [comment]
If for any reason this doesn't go through I'll make a separate PR, gimme a break tho I'm lazy :cry:

**Comment by RealestEstate** [comment]
can the spells be bound to macros? It would kinda suck if not

**Comment by ghost** [comment]
Currently no but I can add that feature.

**Comment by RealestEstate** [comment]
appreciated. binding warp to a macro would just make it easier to move. less clicking back and forth.

**Comment by DamianX** [comment]
when is this scheduled to go on the black market?
also what sort of brain damage inspires your commit messages? do you just copy the output of git status?

**Comment by boy2mantwicethefam** [comment]
>also fixes borg targeting

Absolutely based

**Comment by ghost** [comment]
That is the message when no commit name is chosen.

**Comment by ghost** [comment]
upvoted because its cool. 😎

**Pull request merged**